teloxide_tests/src/mock_bot.rs

```diff
@@ -134,6 +136,10 @@ where
         let current_update_id = AtomicI32::new(42);
         let state = Arc::new(Mutex::new(State::default()));
 
+        #[cfg(feature = "parallel")]
+        let lock = None;
```

Comment:
Is it possible for you to instead remove the lock completely from the bot and implement `Clone`? So it could be shared between threads safely